### PR TITLE
Address verification script

### PR DESCRIPTION
**Pull request by DonDuala**
Used in creating the airdrop list

### CODE REVIEW

File: trading_report/src/verify.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+async function verify() {
+  console.log('Verify addresses are valid', new Date().toUTCString());
+
+  const data = parseCSVFromFile("./address.csv");
+  let verifiedAddress = [];
+  for(const address of data){
+    try{
+      const pubKey = new PublicKey(address);
+      if (PublicKey.isOnCurve(pubKey)) {
```
Comment (brittcyr) [inline trading_report/src/verify.ts:13]: Can we check that it is an active wallet with non-zero SOL in it?

**Review by brittcyr — COMMENTED** [review]
Can you run a linter over this?

Do we need to be checking in the csv files here? Not a big problem, but shouldnt unless we need them version controlled or accessed within the status page

File: trading_report/src/verify.ts (hunk 1 above)
Comment (DonDuala) [reply trading_report/src/verify.ts:13]: We shouldnt care whether it has SOL in it right?

Comment (brittcyr) [reply trading_report/src/verify.ts:13]: Non-zero SOL would just indicate that it is a real user who is able to pay gas, not just some generated keypair. We dont need to check that, but would avoid waste.

Comment (DonDuala) [reply trading_report/src/verify.ts:13]: Would potentially piss off any new Solana user who signed up & I did explicitly say in the form you don't need to have funded the account so holding off. Would be a good potential catch for sybil attackers but already filtered those

**Pull request merged**